### ISSUE TITLE
Permissions do not work

### ISSUE THREAD

**Issue by Flugal** [wyozi/wmc-plus#21]
We have a specially-made staffing system where all of our staff departments are merged into four ULX Ranks: staffaspirant, staff1, staff2, and staff3.
I have it set so any staff can use WMCP's full permissions for event music on our StarWarsRP. However, when trying to use the system, it doesn't work, telling you "Access to 'Permission' Denied."
We don't know what's going on or how to fix this, nor do we have an Error Code.

**Comment by yupi2** [comment]
With [MinAccess](https://github.com/wyozi/wmc-plus/blob/master/lua/wmcp/sh_permissions.lua#L7) set to "admin" ULX/ULib seems to check [`ply:IsAdmin()`](https://github.com/TeamUlysses/ulib/blob/3e3e5fed7fbdaa7884f35330e0f1628948c4ddc8/lua/ulib/shared/cami_global.lua#L359).

If the staff ranks don't aren't considered admins by ULX then it could be the problem.
Do any of the admin/staff/owner ranks have the permissions for WMCP?

**Comment by wyozi** [comment]
This is a bit weird. Assuming the error message comes from https://github.com/wyozi/wmc-plus/blob/master/lua/wmcp/sh_permissions.lua#L55 `WMCP_IfPermissionAsync` would have to be called with 'Permission' as the first string argument.

Are you sure "Access to 'Permission' Denied." is the message you're getting?

**Comment by Flugal** [comment]
I said 'Permission' in quotes as this would be the Permission that it is denying.
For example, Access to Playall Denied

**Comment by wyozi** [comment]
Right, it's possible that your ULX is outdated or for some reason doesn't use CAMI while FAdmin does, in which case FAdmin's permission system overrides that of ULX.
